src/TextInput/TextInput.js

```diff
@@ -341,8 +341,11 @@ function getIcons({
 }
 
 /**
- * TextInput accepts a text value from the user. It supports any of the
- * text-based input types, such as `number` or `email`.
+ * TextInput allows your app to accept a text value from the user. It supports
+ * any of the text-based input types, such as `number` or `email`.
+ *
+ * TextInputs can be composed into other components, such as search fields and dialogs.
+ * Mineral UI provides [FormField](../form-field) to add labeling and accessibility.
```

Comment:
Consider...

TextInput should rarely be used outside of a `[FormField](../form-field)`.

Reply:
I put that content into the Usage section, and I wasn't sure if I should duplicate it up here.